WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -10953,7 +10953,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nrake lint\nelse\necho \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\nrake lint\nelse\necho \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
This is really necessary or is Xcode mocking us?

Reply:
🤦‍♀️I didn't see that get committed. Xcode did this because I _looked_ at the shell script.